include/deal.II/grid/tria_description.h

```diff
@@ -430,6 +430,46 @@ namespace TriangulationDescription
      * dealii::parallel::distributed::Triangulation, where the partitioning is
      * adopted unaltered.
      *
+     * Example for a serial Triangulation:
+     *
+     * @code
+     * Triangulation<dim, spacedim> tria_base;
+     *
+     * // fill serial triangulation (e.g., read external mesh)
+     * GridIn<dim, spacedim> grid_in;
+     * grid_in.attach_triangulation(tria_base);
+     * grid_in.read(file_name);
+     *
+     * // partition serial triangulation
+     * GridTools::partition_triangulation(
+     *   Utilities::MPI::n_mpi_processes(comm), tria_base);
+     *
+     * // create description
+     * const auto description = TriangulationDescription::Utilities::
```

Comment:
Since the type here is the present class, I do not think `auto` is a good idea here and suggest to spell out the type (and similarly in the parallel case below):
```suggestion
     * const TriangulationDescription::Description<dim, spacedim> description
     *   = TriangulationDescription::Utilities::
```